apps/web/src/components/editor/ai-live-preview.tsx

```
@@ -0,0 +1,421 @@
+      cpuUsage: result.performance.cpuUsage,
+      memoryUsage: result.performance.memoryUsage / 1024 / 1024, // MB
+    });
+    
+    // Update overlays if enabled
+    if (overlaysEnabled && overlayCanvasRef.current && videoElement) {
+      drawAIOverlays(result.currentFrame);
+    }
+  }, [overlaysEnabled, videoElement]);
+  
+  // Start/stop analysis based on visibility and video element
+  useEffect(() => {
```

Biome (2.1.2)

[error] This hook does not specify its dependency on overlaysEnabled.

This dependency is being used here, but is not specified in the hook dependency list.

Unsafe fix: Add the missing dependency to the list.


(lint/correctness/useExhaustiveDependencies)